### DISCUSSION TITLE
lost TCM

### DISCUSSION THREAD

**Discussion by givemefreetv** [iptv-org/community#1085] [Q&A]
I was using this playlist https://iptv-org.github.io/iptv/countries/us.m3u and stupidly (in hindsight) updated the playlist through the player.  Now both the TCM East and TCM West channels are gone.  Anyone have another link I can use to get them back.  Thanks

**Comment by GithubDaveW** [comment]
I have no answer, but I have the same question.  Of all of the Github IPTV Channels, I valued the two TCM Channels the most.  If anyone can make them available again (or suggest alternative sources) - it would be incredibly appreciated.  (TY)

**Comment by freearhey** [comment]
This channels has been added to our blocklist due to the claims of the copyright holder: [#16839](https://github.com/iptv-org/iptv/issues/16839)
